src/table/virtualizedDataTable.jsx

```diff
@@ -1191,214 +1499,89 @@ class VirtualizedDataTable extends Component {
       cellWidth: columnWidth,
       cellHeight: tableProps.rowHeight - 2,
     });
-    return _.assign({}, columnProps, {
+    return {
+      ...columnProps,
       columnKey,
       cellRenderer,
       headerRenderer,
       flexShrink: 0,
-      style: styles.column,
+      style: STYLES.column,
       width: columnWidth,
-    });
-  }
-
```

Comment:
This function was inlined below, in the spirit of removing render subfunctions as much as possible